src/error.js

```diff
@@ -361,7 +362,14 @@ export function reportErrorToServerOrViewer(win, data) {
   return maybeReportErrorToViewer(win, data).then((reportedErrorToViewer) => {
     if (!reportedErrorToViewer) {
       const xhr = new XMLHttpRequest();
-      xhr.open('POST', urls.errorReporting, true);
+      // Override the errorReportingUrl to test the new error reporting endpoint.
+      const newErrorReportingUrl =
+        'https://us-central1-amp-error-reporting.cloudfunctions.net/r';
+      const url =
+        IS_ESM || Math.random() < 0.2
```

Comment:
Should that `0.2` be reading from a constant or config somewhere?

Reply:
ill move it up as a constant on this file